scripts/migration/v0.0.9__add-is-hidden-field.ts

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node -r esm -r ts-node/register
+
+import { MigrateOptions } from "fireway";
+
+export const migrate = async ({ firestore }: MigrateOptions) => {
+  console.log(`Updating worlds...`);
+
+  const { docs: worldDocs } = await firestore.collection("worlds").get();
+
+  for (const worldDoc of worldDocs) {
+    if (Object.keys(worldDoc.data()).indexOf("isHidden") !== -1) {
```

Comment:
```suggestion
    if ("isHidden" in worldDoc.data()) return;
```

Reply:
Of course. I was knee deep in firestore rules which doesn't allow this :(

Reply:
Fixed in ea34b8ac

Reply:
Major difference between `in` which goes up the prototype chain and `Object.keys()` which only return own keys. The only suggestion I'd make here was to use `.includes()` instead of `indexOf`